dist/gridstack.js

```diff
@@ -800,6 +800,9 @@
                         .removeData('draggable')
                         .removeClass('ui-draggable ui-draggable-dragging ui-draggable-disabled')
                         .unbind('drag', onDrag);
+                    if(el.find('div.ui-resizable-handle').length >0){
+                        el.find('div.ui-resizable-handle').remove()
```

Comment:
this for me fixes the issue where dragging a griditem to a second grid causes double style preventing resizing of it afterward.